### PR TITLE
chore: Included githubactions in the dependabot config

### PR DESCRIPTION
**Pull request by naveensrinivasan**
This should help with keeping the GitHub actions updated on new releases. This will also help with keeping it secure.

Dependabot helps in keeping the supply chain secure https://docs.github.com/en/code-security/dependabot

GitHub actions up to date https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot

https://github.com/ossf/scorecard/blob/main/docs/checks.md#dependency-update-tool
Signed-off-by: naveensrinivasan <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

**Comment by kasif-adnan** [comment]
@naveensrinivasan , could you fix https://github.com/test-kitchen/test-kitchen/runs/6322482061?check_suite_focus=true

File: .github/dependabot.yml [hunk 1]
```diff
@@ -7,3 +7,7 @@ updates:
       interval: "daily"
       time: "06:00"
       timezone: America/Los_Angeles
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule: 
+      interval: daily
```
Comment (damacus) [inline .github/dependabot.yml:13]:
```suggestion
      interval: weekly
```

For sanity 😆

File: .github/dependabot.yml [hunk 2]
```diff
@@ -7,3 +7,7 @@ updates:
       interval: "daily"
       time: "06:00"
       timezone: America/Los_Angeles
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
```
Comment (kasif-adnan) [inline .github/dependabot.yml:12]: trailing space here is causing yaml lint to fail

**Review by kasif-adnan — CHANGES_REQUESTED** [review]

**Review by kasif-adnan — APPROVED** [review]

**Comment by naveensrinivasan** [comment]
Do you need anything else to merge this? Thanks

**Pull request merged**